src/plugins/tracking/util/clickState.js

```diff
@@ -0,0 +1,56 @@
+import JSCookie from 'js-cookie';
+
+/*
+ * This module provides utilities for managing click tracking data in Redux
+ * state an saving that data to a browser cookie
+ */
+
+export const COOKIE_NAME = 'click-track'; // must remain in sync with Meetup Classic implementation
+
+const BrowserCookies = JSCookie.withConverter({
+	read: (value, name) => value,
```

Comment:
We should drop `name` from here and from `write` since we don't use that variable.